### DISCUSSION TITLE
Is there a way to disable Dark Reader on a specific element?

### DISCUSSION THREAD

**Discussion by KamilDev** [darkreader/darkreader#13663] [Q&A]
I know it's possible to disable Dark Reader on the entire website using:
```js
const lock = document.createElement('meta');
lock.name = 'darkreader-lock';
document.head.appendChild(lock);
```
Is there anything that would disable Dark Reader from recoloring a specific element?

**Comment by e-t-l** [comment]
Yes. In the Dark Reader flyout, click Settings > Advanced > Dev Tools > Inversion Fix Editor.

**Reply by Osirisxblack** [reply]
yeahhhhh👍
